www_src/components/basic-element/basic-element.jsx

```diff
@@ -115,113 +137,73 @@ var El = React.createClass({
     }
   },
 
-  render: function() {
-    // FIXME: TODO: this should be this.state - getInitialState() is purely for
-    //              getting the state of a component prior to mounting, we've
-    //              overloaded it to do somehow it was never intended to do.
-    var state = this.getInitialState();
-
-    var className = classes('el', 'el-' + this.props.type, {
-      touchactive: this.state.touchactive,
-      current: this.props.isCurrent
-    });
+  generateDestinationButton: function() {
+    return (
+      <div className="el-container" key={this.props.key + '-2'}>
+        {/* using onTouchEnd because onClick doesn't work for some reason...touchhandler.js preventing it? */}
```

Comment:
Might be if touchhandler.js prevents default on any events, that would cancel onClick